charts/ks-devops/templates/ingress.yaml

```diff
@@ -10,7 +9,7 @@ kind: Ingress
 metadata:
   name: {{ $fullName }}
   labels:
-    {{- include "ks-devops.labels" . | nindent 4 }}
+    {{- include "ks-devops-apiserver.labels" . | nindent 4 }}
```

Comment:
Why should we change "ks-devops.labels" to "ks-devops-apiserver.labels"?

Reply:
IMO, only the front-end needs the ingress. So the apiserver does not need it.

Reply:
> Why should we change "ks-devops.labels" to "ks-devops-apiserver.labels"?

Without this 'ks-devops.labes'